provider/lxd/credentials.go

```diff
@@ -166,27 +171,19 @@ func (p environProviderCredentials) readOrGenerateCert(logf func(string, ...inte
 
 // FinalizeCredential is part of the environs.ProviderCredentials interface.
 func (p environProviderCredentials) FinalizeCredential(ctx environs.FinalizeCredentialContext, args environs.FinalizeCredentialParams) (*cloud.Credential, error) {
-	var interactive bool
+	// TODO (stickupkid): if there are no server certs for the following, use
+	// the lxd API
 	switch authType := args.Credential.AuthType(); authType {
 	case interactiveAuthType:
```

Comment:
I don't think this is quite right.

I think the options presented to the user should be:
- certificate
- certificate from files

Then when creating the credential from files, the actual certs are written as a credential of type "certificate".

Reply:
So I've changed this in https://github.com/juju/juju/pull/8879/commits/a14f2ee085e213e9786bdfce2af90deff98e53a6